website/docs/sdk-reference/dotnet.md

```diff
@@ -57,29 +66,48 @@ client.Dispose();
 - caching your setting values and feature flags.
 - serving values quickly in a failsafe way.
 
-`new ConfigCatClient()` creates a client with default options.
+`ConfigCatClient.Get(sdkKey: "<sdkKey>")` returns a client with default options.
+
+The `Get` method has an optional callback parameter, which can be used to configure the client via a `ConfigCatClientOptions` object:
```

Comment:
Please try to avoid using the word `configure` in any documentation. Because it collides with many things that are called config, configure, and configuration in the ConfigCat ecosystem.